catboost/docs/en/concepts/python-usages-examples.md

```diff
@@ -995,7 +995,7 @@ Examples:
 
 #### {{ error-function--Logit }}
 
-{% include [reusage-common-phrases-implemantation-example](../_includes/work_src/reusage-common-phrases/implemantation-example.md) %}
+{% include [reusage-common-phrases-implementation-example](../_includes/work_src/reusage-common-phrases/implemantation-example.md) %}
```

Comment:
there more ocurences -- please change all of them
```
catboost/docs/en/concepts/python-usages-examples.md:998:{% include [reusage-common-phrases-implemantation-example](../_includes/work_src/reusage-common-phrases/implemantation-example.md) %}
catboost/docs/en/concepts/python-usages-examples.md:1032:{% include [reusage-common-phrases-implemantation-example](../_includes/work_src/reusage-common-phrases/implemantation-example.md) %}
catboost/docs/en/concepts/python-usages-examples.md:1064:{% include [reusage-common-phrases-implemantation-example](../_includes/work_src/reusage-common-phrases/implemantation-example.md) %}
catboost/docs/en/concepts/python-usages-examples.md:1132:{% include [reusage-common-phrases-implemantation-example](../_includes/work_src/reusage-common-phrases/implemantation-example.md) %}
catboost/docs/en/concepts/python-usages-examples.md:1171:{% include [reusage-common-phrases-implemantation-example](../_includes/work_src/reusage-common-phrases/implemantation-example.md) %}
catboost/docs/en/concepts/python-usages-examples.md:1208:{% include [reusage-common-phrases-implemantation-example](../_includes/work_src/reusage-common-phrases/implemantation-example.md) %}
```

Reply:
please also make sure that you all references to other renamed files -- just do something like this `grep -n implemantation-example.md catboost/docs -r`

Reply:
Thanks! Luckily, it was the only typo with such problem. I scanned all text entries of files again. After the changes, only an error related to my inattention could remain.

Reply:
In project root: 
```rg --files catboost/docs/ | rg -uuu -i 'featrue|implemant|targer|resctric|prdict|reuzin'``` - incorrect file names
```rg -uuu -i 'featrue|implemant|targer|resctric|prdict|reuzin' catboost/docs``` - incorrect text occurrences

Reply:
👍